src/react/components/pages/editorPage/editorSideBar.test.tsx

```diff
@@ -27,89 +27,82 @@ describe("Editor SideBar", () => {
     it("Initializes state without asset selected", () => {
         const props: IEditorSideBarProps = {
             assets: testAssets,
-            onAssetSelected: onSelectAssetHanlder,
+            onAssetSelected: onSelectAssetHandler,
         };
 
         const wrapper = createComponent(props);
         expect(wrapper.state().selectedAsset).not.toBeDefined();
+        expect(wrapper.state().scrollToIndex).toBe(0);
     });
 
     it("Initializes state with asset selected", () => {
+        const selectedAssetIndex = 3;
+
         const props: IEditorSideBarProps = {
             assets: testAssets,
-            selectedAsset: testAssets[0],
-            onAssetSelected: onSelectAssetHanlder,
+            selectedAsset: testAssets[selectedAssetIndex],
+            onAssetSelected: onSelectAssetHandler,
         };
 
         const wrapper = createComponent(props);
         expect(wrapper.state().selectedAsset).toEqual(props.selectedAsset);
+        expect(wrapper.state().scrollToIndex).toBe(selectedAssetIndex);
     });
 
     it("Updates states after props have changed", (done) => {
         const props: IEditorSideBarProps = {
             assets: testAssets,
             selectedAsset: null,
-            onAssetSelected: onSelectAssetHanlder,
+            onAssetSelected: onSelectAssetHandler,
         };
 
         const wrapper = createComponent(props);
+
+        const selectedAssetIndex = 3;
         wrapper.setProps({
-            selectedAsset: testAssets[0],
+            selectedAsset: testAssets[selectedAssetIndex],
         });
 
         setImmediate(() => {
-            expect(wrapper.state().selectedAsset).toEqual(testAssets[0]);
+            const state = wrapper.state();
+            expect(state.selectedAsset).toEqual(testAssets[selectedAssetIndex]);
+            expect(state.scrollToIndex).toEqual(selectedAssetIndex);
+
             done();
         });
     });
 
-    it("Calls onAssetSelected handler when an asset is selected", (done) => {
```

Comment:
removed this test because there isn't a way to access the asset element inside a react-virtualized list.  Without this element, we cannot simulate a click, therefore cannot test whether onAssetSelected is called.